### ISSUE TITLE
oe_translation_post_update_0002() fails if oe_translator role was deleted on website

### ISSUE THREAD

**Issue by donquixote** [openeuropa/oe_translation#245]
If on a website the oe_translator role no longer exists, the post update hook will fail.

```php
function oe_translation_post_update_0002(&$sandbox = NULL) {
  $role = Role::load('oe_translator');
  $role->grantPermission('accept translation request');
  $role->grantPermission('sync translation request');
  $role->save();
}
```

> Error: Call to a member function grantPermission() on null in oe_translation_post_update_0002()

**Comment by donquixote** [comment]
An alternative could be to assign these new permissions to any role that currently has another permission.
But I don't know if this can be safely assumed to be the right thing for all websites. And I don't know which prior permission we would use for this.

**Comment by upchuk** [comment]
@donquixote Thanks. Will be fixed in alpha2. We cannot assign to a random role. So we will just bail out if the role is missing.

**Comment by donquixote** [comment]
Would be nice to have something we can use as a patch.

**Comment by upchuk** [comment]
@donquixote It's fixed here: https://github.com/openeuropa/oe_translation/pull/247
You can use this branch as patch

**Comment by donquixote** [comment]
> @donquixote It's fixed here: #247 You can use this branch as patch

There is another one that needs fixing for `oe_translation_local_install()`.

**Comment by upchuk** [comment]
Indeed. Fixed as well
